msg/AckermannDriveSetpoint.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp # time since system start (microseconds)
+
+float32 speed # [m/s] requested rover ground speed
```

Comment:
Suggestion: 

Try to keep this consistent with the current rover message structure, where speed is the speed_actual in this case and the speed setpoint should try to have a similar naming convention as the differential drive rover. 

https://github.com/PX4/PX4-Autopilot/blob/6e4d80776c1b48e72f4cc4c634b14c3bfc91e829/msg/DifferentialDriveSetpoint.msg#L3